src/assets/styles/variables.less

```diff
@@ -18,4 +18,4 @@
 @expanded-width: 300px;
 
 // Heights
-@splitter-height: 2px;
\ No newline at end of file
+@splitter-height: 1px;
```

Comment:
Ok, will revert then. His design definitely has 1px splitter though (on the pdfs)